README.md

```diff
@@ -18,7 +18,7 @@ be used in long running processes.
 The library is maintained by [Cloudflare](https://www.cloudflare.com) and
 [Cilium](https://www.cilium.io). Feel free to
 [join](https://cilium.herokuapp.com/) the
```

Comment:
I think while we're here cilium.herokuapp.com could be replaced by ebpf.io/slack. While it seems to just be a redirect at the moment, it's probably what cilium wants to be here.

Reply:
Yep, feel free to change the join link as well, thanks :+1:

Reply:
done